### PR TITLE
Fix size

### PR DESCRIPTION
**Pull request by vyorkin**
refs #4 @unindented I've just noticed, actually [it uses relative path](https://github.com/webpack/webpack/blob/master/lib/Stats.js#L165). So the only thing I've fixed is the `size` ouput

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -15,12 +15,14 @@ StatsPlugin.prototype.apply = function apply (compiler) {
   var options = this.options
 
   compiler.plugin('emit', function onEmit (compilation, done) {
+    const source = JSON.stringify(compilation.getStats().toJson(options));
```
Comment (unindented) [inline index.js]: Extra-semicolon

File: index.js [hunk 2]
```diff
@@ -15,12 +15,14 @@ StatsPlugin.prototype.apply = function apply (compiler) {
...
+
     compilation.assets[output] = {
       size: function getSize () {
-        return 0
+        return source.length;
```
Comment (unindented) [inline index.js]: Extra-semicolon

File: index.js [hunk 3]
```diff
@@ -15,12 +15,14 @@ StatsPlugin.prototype.apply = function apply (compiler) {
...
       },
       source: function getSource () {
-        return JSON.stringify(compilation.getStats().toJson(options))
+        return source;
```
Comment (unindented) [inline index.js]: Extra-semicolon

**Comment by unindented** [comment]
@vyorkin Awesome!

If you remove the extra-semicolons (I'm linting using [standard](https://github.com/feross/standard)) I'll merge this right away.

**Comment by vyorkin** [comment]
done

**Comment by unindented** [comment]
It looks like there's a unit test now failing: https://travis-ci.org/unindented/stats-webpack-plugin/jobs/94395990

Could you look into it?

**Comment by vyorkin** [comment]
oops, sorry, sure, give me a minute

**Comment by vyorkin** [comment]
oh, that looks so weird:
```
             ]
             "chunks": [
               0
             ]
      +      "emitted": true
             "name": "bundle.js"
             "size": 1622
           }
      +    {
      +      "chunkNames": []
      +      "chunks": []
      +      "name": "stats.json"
      +      "size": 0
      +    }
```
but all I did is just moved `JSON.stringify...` out of the function scope..

**Comment by vyorkin** [comment]
Yes, I think now it behaves correctly, I just need to fix the failing test

**Comment by vyorkin** [comment]
I don't know whats wrong, seriously, I'll take a look at it later, it isn't worth it right now

**Comment by unindented** [comment]
Fixed with https://github.com/unindented/stats-webpack-plugin/commit/2643278d62fbb7f5edb6371b245b17ce1680c335.

**Comment by vyorkin** [comment]
cool! sorry, I didn't have time to do it myself :(

**Pull request closed without merging**